guides/source/autoloading_and_reloading_constants_classic_mode.md

```diff
@@ -482,8 +482,6 @@ What is described above are the defaults with a newly generated Rails app. There
 
 See also [Autoloading in the Test Environment](#autoloading-in-the-test-environment).
 
-`config.autoload_paths` is not changeable from environment-specific configuration files.
-
```

Comment:
I couldn't find a reason this was the case?

Reply:
Probably the docs were describing how things worked. I'd bet this is an overlook and just a consequence of how the initializers were ordered.